README.md

```diff
@@ -22,8 +22,17 @@ As you embark on these exercises, you're set to learn key practices in modern ap
 
 By the end of these exercises, you'll have a solid foundation in deploying and managing Kubernetes applications using Argo CD.
 
+> Hint to advanced topics that are beneficial to expert users. Like:
```

Comment:
this should be towards the end right? Probably better if the beginning is not too cluttered right?

Reply:
Yeah, maybe just as a follow up. Anyway I wanted to point out that the Kutespace is not only useful for novices but for any knowledge level. It can be used as an "Argo CD dev/staging environment".